agent/wsclient/client.go

```diff
@@ -35,10 +35,11 @@ import (
 	"time"
 
 	"github.com/aws/amazon-ecs-agent/agent/config"
-	"github.com/aws/amazon-ecs-agent/agent/utils"
-	"github.com/aws/amazon-ecs-agent/agent/utils/cipher"
 	"github.com/aws/amazon-ecs-agent/agent/wsclient/wsconn"
```

Comment:
I do not think it makes sense to switch just wsconn imports to ecs-agent/wsclient. The plan is to plug in the ecs-agent/wsclient with ecs-agent/tcs and ecs-agent/acs, and once we have the confidence in testing, we would completely remove the agent/wsclient package. It made sense to move the utils to avoid the cyclic dependency issue between agent and ecs-agent.

Reply:
I was just thinking since the files in `agent/wsclient/wsconn` and `ecs-agent/wsclient/wsconn` are identical, we could just have a single one living in `ecs-agent/`to refer to as a source of truth since we will be removing `agent/wsclient/wsconn` in the future anyway.

But I don't feel super strongly about this, your plan achieves the same end result and works for me.